doom2016.asl

```diff
@@ -185,18 +188,31 @@ state("DOOMx64vk", "6, 1, 1, 818") {
 }
 
 // 2018-03-29 Patch (Latest)
+state("DOOMx64", "6, 1, 1, 321") {
+    float bossHealth: 0x036F9E08, 0x30, 0x4D0, 0x420, 0x2F8, 0x20, 0x2F0, 0x1B4;
+    bool start: 0x427EA00;
+    bool canStart: 0x2E4A378;
+    bool finalHit: 0x3E59794;
+    bool isLoading: 0x41434D9;
+
+    string35 mapName: "tier0_s64.dll", 0x58180, 0x17;
+    string60 mapFile: 0x415FE15;
+}
+
 state("DOOMx64vk", "6, 1, 1, 321") {
     float bossHealth: 0x307EF08, 0x2CD80, 0x2B78;
     bool start: 0x597FCD0;
     bool canStart: 0x56BF5D8;
     bool finalHit: 0x5557504;
-    bool isLoading: 0x5845a29;
+    bool isLoading: 0x5845A29;
```

Comment:
Was this supposed to be changed?

Reply:
Yes, I changed the capitalization to be in line with the rest of the file.